apiary.apib

```diff
@@ -3290,7 +3290,7 @@ attributes and information about the containing bucket.
 ### Add Column to Table [POST]
 Adds a new column to an existing table. This request is [asynchronous](#introduction/synchronous-and-asynchronous-calls).
 
-*Attribute definition is allowed (and required) for typed tables created via [table definition](#reference/tables/create-table-definition)* with Synapse or Exasol backend only
+*Attribute definition and basetype is allowed (and required) for typed tables created via [table definition](#reference/tables/create-table-definition)* with Synapse, Snowflake and Exasol backend only
```

Comment:
Tady to chce upresnit, pac ted je podpora basetype u pridani sloupce jen pro Snowflake.
Synapse a Exasol to jeste nemaji.

Reply:
Majů to aj Exasol aj Synapse